### DISCUSSION TITLE
FR : HttpClient to RequestHandler

### DISCUSSION THREAD

**Discussion by mw-nfi** [LuckyPennySoftware/MediatR#747] [Q&A]
Hello,
I am very enthusiastic about Mediatr and have recently used the processors. But now I have no real error but rather a function request. How could one automatically make a Http request to the Api to use the desired response in the handler? Sure, I could always create the client. But is there a way to create a request through the processor and release the result in the handler?

Thanks a lot!

**Comment by jbogard** [comment]
I'm not sure how possible/feasible (or desirable) this would be. To make an HTTP request you still need the URL (or with typed clients, the method to call). How would that be done automatically?

**Reply by mw-nfi** [reply]
Yes, that's right. I have already thought about this. Since there could possibly be other requests, one would have to distinguish these. Thus one could define the missing information in the respective inquiry. It might not look so clean, but you would have the information in Theo same class as request. Or definable via options.
